app/js/components/questions/new/Index.react.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import * as QuestionActions from '../../../actions/QuestionActions.js';
+import NewQuestionPage from './NewQuestionPage.react';
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+import toastr from 'toastr';
+
+class NewQuestion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
Actually this is a container component. It's valid the way it is. The state is peculiar to the component. It represent the question being created.